tests/e2e/device/Makefile

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.

```

Comment:
would this file get replaced with CMake or leave it as Make?

Reply:
Yes, we will change it to cmake... once we have root cmake support.